core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletMetadataBuilder.java

```diff
@@ -244,9 +245,17 @@ public TabletMetadataBuilder deleteHostingRequested() {
   }
 
   @Override
-  public TabletMetadataBuilder putOperation(TabletOperationId opId) {
+  public TabletMetadataBuilder startOperation(TabletOperationId opId) {
```

Comment:
I renamed this because the `internalBuilder.startOperation` method does two things.

Reply:
I reverted this in a later commit